doc/sources/tutorials/crashcourse.rst

```diff
@@ -0,0 +1,55 @@
+.. _crashcourse:
+
+Crash Course
+============
+
+Crash Course(or Kivy Crash Course) is a set of 14 video tutorials by Kivy core
+developer `inclement <https://github.com/inclement>`_ which provides a simple
+walkthrough in Kivy for users who know how to code in Python, but is also
+friendly even to beginners with Python as with Kivy. After Pong and Paint
+tutorials, this set of videos covers more features and tweaks that can be used
+to create your app quicker, your code elegant and eye-friendly.
+
+
+Basic Info
```

Comment:
Could `Basic Info`, `What the Crash Course covers` and `Let's start` be removed? The course is an external resource and we do not track if the content and number of videos and articles change, so these sections will just become outdated and incorrect.

The playlist link could be simply added to the `Links` section.